README.md

```diff
@@ -1767,6 +1767,76 @@ function Squash-Commits {
 }
 ```
 
+# Adding code to other people's repositories.
```

Comment:
Needs to be added to the TOC. I'm also not sure that this should be a top-level entry.